src/dotnet/APIView/APIViewTest/CodeFileRendererTests.cs

```diff
@@ -377,7 +377,7 @@ public void Render_WithInvalidId_ReturnsValidIdNames(string invalidId, string va
             // Assert
             Assert.Single(result);
             Assert.Equal(" KeywordLine_One:", result[0].DisplayString);
-            Assert.Equal(validId, result[0].ElementId);
+            //Assert.Equal(validId, result[0].ElementId);
```

Comment:
is there a tracking item to enable this test/re-write them again?
Does this mean that by fixing the bug, the functionality that depended on the sanitization of the ID is now broken?